python/pyarrow/includes/libarrow.pxd

```diff
@@ -366,6 +366,7 @@ cdef extern from "arrow/api.h" namespace "arrow" nogil:
         shared_ptr[CDevice] device()
         const shared_ptr[CMemoryManager] memory_manager()
         CDeviceAllocationType device_type()
+        CResult[shared_ptr[CBuffer]] Copy(shared_ptr[CBuffer] source, const shared_ptr[CMemoryManager]& to) const
```

Comment:
In C++, this method is `static` instead of `const`. Any idea if Cython needs to reflect this to match exactly?

https://github.com/apache/arrow/blob/ab0a40ee34217070f14027776682074c55d0b507/cpp/src/arrow/buffer.h#L331

Reply:
Oh good catch. How do I declare static? Is it correct that I declare it by adding a `@staticmethod` header above line 369? @jorisvandenbossche

Reply:
Ah, good catch. It does seem to work right now as well, but so how we do it for other static methods is to decorate it here with `@staticmethod` and then in the code use it as such (`CBuffer.Copy(self.buffer, ...)`)

See for example `CTable` `Make` method in this file.